src/apis/notice-post-comments/controllers/notice-post-comments.controller.ts

```diff
@@ -40,7 +40,11 @@ export class NoticePostCommentsController {
     private readonly noticePostCommentsService: NoticePostCommentsService,
   ) {}
 
-  @ApiNoticePostComment.Create({ summary: '공지 게시글 댓글 생성' })
+  @ApiNoticePostComment.Create({
+    summary: '공지 게시글 댓글 생성',
+    description:
+      'path가 가리키는 공지 게시글이 댓글을 비 허용할 시 403 에러 발생',
```

Comment:
```suggestion
    description:
      'path가 가리키는 공지 게시글이 댓글을 비 허용할 시 403 에러 발생',
```
description은 필요없을듯합니다. 차라리 에러로 반환하는 메시지를 보고 가이드를 주는게 나을거같아요.
모든 api 스펙을 description에 나열할수는 없으니까요